sdv/data_processing/data_processor.py

```diff
@@ -314,6 +314,10 @@ def update_transformers(self, column_name_to_transformer):
 
         self._hyper_transformer.update_transformers(column_name_to_transformer)
 
+    def _create_hyper_transformer_instance(self):
```

Comment:
why aren't we just using the `HyperTransformer` in the `DataProcessor`? We shouldn't need to make another one